osfoffline/application/background.py

```diff
@@ -89,7 +91,7 @@ def start_folder_observer(self):
         self.observer = Observer()  # create observer. watched for events on files.
         # attach event handler to observed events. make observer recursive
         self.observer.schedule(self.event_handler, self.osf_folder, recursive=True)
-        # LocalDBSync(self.user.osf_local_folder_path, self.observer, self.user).emit_new_events()
+        LocalDBSync(self.user.osf_local_folder_path, self.observer, self.user).emit_new_events()
```

Comment:
If I'm reading this right, this doesn't just re-index when we restart the application. It also does a full re-index every time we click sync now, and every time the preferences panel is opened/closed. Thus turning this on will require regression testing of *all* sync events, not just those that happen while the application is closed.

This line appears in `start_folder_observer(...)`, which is called in `BackgroundWorker.run(...)`, which is called when the thread is started. Eg:
1. `OsfApp.start` (which is our intended goal)
2. `OsfApp.resume` (prefs panel close, syc now, etc)